CarFleetManagement/api/README_LYNX_AUTH.md

```
@@ -0,0 +1,202 @@
+```
+
+### Validate Token
+
+**Endpoint:** `/api/auth/validate-token/`
+
+**Method:** GET
+
+**Description:** Check if a token is valid
+
+**Headers:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)